data/json/recipes/basecamps/recipe_primitive_field.json

```diff
@@ -921,6 +921,7 @@
     "never_learn": true,
     "construction_blueprint": "faction_base_field_camp_blacksmith_0",
     "blueprint_name": "Blacksmithy survey",
+    "blueprint_requires": [ { "id": "faction_base_field_camp_blacksmith_1" } ],
```

Comment:
When did this lose
```
    "blueprint_requires": [ { "id": "not_an_upgrade" } ]
```
ah, in c2f72840.

Could you just revert all of c2f72840 for data/json/recipes/basecamps?